src/groups/mqb/mqbblp/mqbblp_clusterproxy.cpp

```diff
@@ -749,7 +749,7 @@ void ClusterProxy::processEvent(const bmqp::Event&   event,
                        << "Received unexpected event: " << event;
         BSLS_ASSERT_SAFE(false && "Unexpected event received");
         return;  // RETURN
-    }            // break;
+    }  // break;
```

Comment:
There seem be only formatting changes in this file

Reply:
Yeah looks so, clang-format must have picked it up at some point. Should I just discard these changes?

Reply:
The CI formatting check complains about these changes, but we might use an older version of clang-format which doesn't handle these cases well. I will look how if we can update it.

Reply:
Yes, there is a [mismatch](There is a [mismatch](https://github.com/bloomberg/blazingmq/actions/runs/9998715055/job/27660062596?pr=352#step:3:14) between Jenkins formatter and the one in the )